compose/metrics/client.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+
+import requests
+from docker import ContextAPI
+
+
+class Status(Enum):
+    SUCCESS = "success"
+    FAILURE = "failure"
+    CANCELED = "canceled"
+
+
+class MetricsSource:
+    CLI = "docker-compose"
+
+
+METRICS_PORT = 9000
```

Comment:
We need to run over sockets, rather than ports, and named pipes on windows. 
Desktop has 2 defaults :
* `\\.\pipe\docker_cli`  on windows
* `/var/run/docker-cli.sock` on mac

e2e tests can have another socket in a test tmp folder, so we ensure this works over sockets rather than ports. Will need to validate on windows how this works

Reply:
Done. Thanks!